I18N.md

```diff
@@ -0,0 +1,27 @@
+## i18n
+
+Internationalization is implemented with [i18next](https://www.i18next.com/) and
+[react-i18next](https://react.i18next.com/). The i18n namespace must be declared in
+TRANSLATION_NAMESPACE property to avoid naming conflicts.
```

Comment:
```suggestion
[react-i18next](https://react.i18next.com/). The i18n namespace must be declared at build-time in
an environment variable called: `TRANSLATION_NAMESPACE` in order to avoid naming conflicts.
```

Reply:
changes done